pkg/karmadactl/get.go

```diff
@@ -345,6 +345,20 @@ func (g *CommandGetOptions) checkPrintWithNamespace(mapping *meta.RESTMapping) b
 func (g *CommandGetOptions) getObjInfo(wg *sync.WaitGroup, mux *sync.Mutex, f cmdutil.Factory,
 	cluster string, objs *[]Obj, allErrs *[]error, args []string) {
 	defer wg.Done()
+
+	restClient, err := f.RESTClient()
+	if err != nil {
+		*allErrs = append(*allErrs, err)
+		return
+	}
+
+	// check if it is authorized to proxy this member cluster
```

Comment:
I add that checking if it is authorized to proxy member clusters here. How do you think? @XiShanYongYe-Chang

Reply:
Feels good.